bonsai/BudgetTransactions/TransactionCell.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  TransactionCell.swift
+//  bonsai
+//
+//  Created by hoang on 08.12.2021.
+//
+
+import SwiftUI
+
+struct TransactionCell: View {
+    
+    let item: Transaction
+    
+    init(item: Transaction) {
+        self.item = item
+    }
+    
+    var body: some View {
+        HStack {
+            Image(uiImage: item.category.icon)
+                .renderingMode(.template)
+                .foregroundColor(Color(item.category.color))
+            Text(item.category.title)
+                .font(BonsaiFont.body_17)
+                .foregroundColor(Color.white)
+            Spacer()
+            Text("\(item.amount)")
```

Comment:
Missing "+"/ "-" sign for income/expence

Reply:
well no "+" actually, as we probably only should display expenses here
That also means we should filter transactions and only display expenses on BudgetTransactionView

Reply:
got you, but i have problem when i try to get access from `Type` of Transaction - syntax compile error
for example in condition 
if transaction.type != .expense { } compile error

Reply:
use if case syntax

Reply:
if case .expence == transaction.type

Reply:
Btw I think this `type` check should be done in init where you create the dictionary

Reply:
I mean we only show expenses in this view anyway, so we don't need to check type of transaction right in this view, because it is supposed to only receive expense transactions anyway

Reply:
fixed